R/PackageFunctionReporter.R

```diff
@@ -13,9 +13,10 @@
 #'                 previously-set package name, any cached data will be removed.}
 #'             \item{\bold{Args:}}{
 #'                 \itemize{
-#'                 \item{\bold{\code{pkg_name}}: String with the name of the package}
-#'                 \item{\bold{\code{pkg_path}}: Optional path to the source code. 
-#'                     To be used for test coverage, if provided.}
+#'                 \item{\bold{\code{pkg_name}}: String with the name of the package.}
+#'                 \item{\bold{\code{pkg_path}}: Optional directory path to source
```

Comment:
after reading this for the third time, I'm thinking we would benefit from using `inheritParams` to centralize docs for these common parameters. I'll throw up an issue

Reply:
added #81